tdp/cli/commands/plan/edit.py

```diff
@@ -0,0 +1,220 @@
+# Copyright 2022 TOSIT.IO
+# SPDX-License-Identifier: Apache-2.0
+
+import logging
+import os
+import re
+import tempfile
+from contextlib import contextmanager
+from typing import Optional
+
+import click
+
+from tdp.cli.queries import get_planned_deployment_log
+from tdp.cli.session import get_session
+from tdp.cli.utils import collections, database_dsn
+from tdp.core.models.deployment_log import DeploymentLog
+
+logger = logging.getLogger("tdp").getChild("edit")
+
+
+def _get_header_message(deployment_log_id: int, temp_file_name: str) -> str:
+    """Returns the header message to be displayed in the temporary file."""
+    return f"""\
+# ------------------- DEPLOYMENT PLAN EDITOR --------------------------
+# Modify this file to edit the deployment plan.
+# Lines beginning with '#' are comments and will be ignored.
+#
+# Current plan id: {deployment_log_id}
```

Comment:
beter to change "plan id" to "planned deployment ID"

Reply:
Done

Reply:
Done